scripts/MultiPlotting/multi_plotting_widget.py

```diff
@@ -11,31 +11,41 @@
 from MultiPlotting.subplot.subplot import subplot
 from MultiPlotting.QuickEdit.quickEdit_widget import QuickEditWidget
 from MultiPlotting.multi_plotting_context import *
+from mantidqt.MPLwidgets import FigureCanvasQTAgg as FigureCanvas
+from matplotlib.figure import Figure
 
+from mantidqt.plotting.ads_observer import FigureManagerADSObserver
 
-class MultiPlotWindow(QtWidgets.QMainWindow):
-    windowClosedSignal = QtCore.Signal()
 
-    def __init__(self, window_title="plotting"):
-        super(MultiPlotWindow, self).__init__()
+class MultiPlotWindow(object):
+    def __init__(self, window_title="plotting", close_callback=None):
+        self.figure = Figure()
+        self.figure.set_facecolor("none")
+        self.canvas = FigureCanvas(self.figure)
+        self.close_callback = close_callback
+
         self.plot_context = PlottingContext()
-        self.multi_plot = MultiPlotWidget(self.plot_context, self)
-        self.setCentralWidget(self.multi_plot)
-        self.setWindowTitle(window_title)
+        self.multi_plot = MultiPlotWidget(window_title, self.plot_context, self.canvas)
```

Comment:
should we just replace `self.multi_plot` with `self.window`?

Reply:
In the medium term we should, I left it with both to minimize the chances of breaking existing code which relies on the old name but having thought a bit more that probably only happens in a couple of places so can probably just update now.